addons/mail/static/src/models/thread_view_topbar.js

```diff
@@ -692,6 +699,9 @@ registerModel({
         pendingThreadDescription: attr({
             default: "",
         }),
+        shouldDisplayDebugIcon: attr({
+            compute: '_computeShouldDisplayDebugIcon',
```

Comment:
Note that you can now write the compute function directly inside the field definition, see the field `thread` on `Channel` for an example. I don't really have a strong preference on this matter though, so it's all up to you to decide whether to move the compute here or leave it where it is now.